Source/Fluxor/DependencyInjection/FluxorOptions.cs

```diff
@@ -14,6 +14,17 @@ public class FluxorOptions
 		internal AssemblyScanSettings[] AssembliesToScan { get; private set; } = Array.Empty<AssemblyScanSettings>();
 		internal Type[] TypesToScan { get; private set; } = Array.Empty<Type>();
 		internal Type[] MiddlewareTypes = Array.Empty<Type>();
+		/// <summary>
+		/// The Lifecycle that should be used when registering Fluxor features/reducers/effects/middleware</br>
+		/// (default) LifecycleEnum.Scoped = Create a new instance for each new request</br>
+		/// LifecycleEnum.Singleton = Create a new instance on first request and reuse for rest of application lifetime</br>
+		/// </br>
+		/// NOTE: indicating Singleton should be done only for exceptional cases.
+		/// For example, in MAUI/Blazor hybrid applications, the main MAUI application is a different scope then each BlazorWebView component
+		/// and state needs to be shared across all scopes of the application
```

Comment:
You can take these outside of `<summary>` and wrap them in `<remarks>`

Reply:
improved xml code docs - let me know if it needs more adjustment